src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -807,6 +886,13 @@ public virtual bool Disconnect(DisconnectParams disconnectParams)
                 return false;
             }
 
+            // This clears the uri of the connection from the tokenUpdateUris map, which is used to track
+            // open editors that have requested a refreshed AAD token.
+            if (info.ConnectionDetails.AuthenticationType == "AzureMFA")

```

Comment:
Probably don't need this check - if a connection is disconnected we can always just try to remove it from the map since it just won't do anything for non-AAD connections